tools/swift-ide-test/swift-ide-test.cpp

```diff
@@ -4473,6 +4489,33 @@ int main(int argc, char *argv[]) {
     }
   }
 
+  for (auto path : options::PluginPath) {
+    InitInvok.getSearchPathOptions().PluginSearchPaths.push_back(path);
+  }
+  if (!options::LoadPluginLibrary.empty()) {
+    std::vector<std::string> paths;
+    for (auto path: options::LoadPluginLibrary) {
+      llvm::errs() << "LoadPluginLibrary: " << path << "\n";
```

Comment:
Was this just debug output during development or do you want to keep it for the future?

Reply:
Argh, I will remove this.